elements/pfe-card/pfe-card.ts

```diff
@@ -27,123 +15,64 @@ import style from './pfe-card.scss';
+ * @cssproperty {<color>} --pf-c-card--m-flat--BorderWidth {@default `1px solid #d2d2d2`}
+ * @cssproperty {<color>} --pf-c-card--m-plain--BoxShadow {@default `none`}
+ * @cssproperty {<color>} --pf-c-card--m-plain--BackgroundColor {@default `transparent`}
+ * @cssproperty {<color>} --pf-c-card--m-rounded--BorderRadius {@default `3px`}
+ * @cssproperty {<color>} --pf-c-card--m-full-height--Height {@default `100%`}
+ * @cssproperty {<color>} --pf-c-card__title--FontFamily {@default `"RedHatDisplayUpdated", helvetica, arial, sans-serif`}
+ * @cssproperty {<color>} --pf-c-card__title--FontSize {@default `1rem`}
+ * @cssproperty {<color>} --pf-c-card__title--FontWeight {@default `700`}
  */
-@customElement('pfe-card') @pfelement()
-export class PfeCard extends LitElement {
+@customElement('pfe-card')
+export class PfeCard extends BaseCard {
   static readonly version = '{{version}}';
 
-  static readonly styles = [style];
+  static readonly styles = [...BaseCard.styles, style];
 
   /**
-   * Optional background image applied to the entire card container.
-   * Alignment of this image can be managed using the `--pfe-card--BackgroundPosition`
-   * variable which is set to `center center` by default.
+   * Optionally provide a size for the card and the card contents.
+   * The default is set to `undefined` and provides default styles.
+   * Compact provides styles which decreases the padding between the sections.
+   * Large provides styles which increases the padding between the sections and the font size for the title, header, and footer.
    */
-  @observed
-  @property({ attribute: 'img-src', reflect: true }) imgSrc?: string;
+  @property({ reflect: true }) size: 'compact' | 'large' | undefined = undefined;
 
   /**
-   * Sets color palette, which affects the element's styles as well as descendants' color theme.
-   * Overrides parent color context.
-   * Your theme will influence these colors so check there first if you are seeing inconsistencies.
-   * See [CSS Custom Properties](#css-custom-properties) for default values
-   *
-   * Card always resets its context to `base`, unless explicitly provided with a `color-palette`.
-   */
-  @colorContextProvider()
-  @property({ reflect: true, attribute: 'color-palette' }) colorPalette?: ColorPalette = 'base';
-
-  /** @deprecated use `color-palette` */
-  @deprecation({ alias: 'colorPalette', attribute: 'color' }) color?: ColorPalette;
+  * Optionally apply a border radius for the drop shadow and/or border.
+  */
+  @property({ type: Boolean, reflect: true }) rounded = false;
 
   /**
-   * Sets color theme based on parent context
-   */
-  @colorContextConsumer()
-  @property({ reflect: true }) on?: ColorTheme;
-
-  /** Optionally adjusts the padding on the container. Accepts: `small`. */
-  @property({ reflect: true }) size?: 'small';
+ * Optionally allow the card to take up the full height of the parent element.
+ */
+  @property({ type: Boolean, reflect: true }) fullHeight = false;
```

Comment:
the attr for this should be `full-height`

Reply:
Ahh yeah that makes sense too, updated & updated styles / demos